projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -415,7 +415,18 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
 
     if (direction !== undefined && !isNaN(offset) && offset !== this._offsetEvent) {
       this._offsetEvent = offset;
-      this.page.emit({ offset });
+      // if scroll was done by mouse drag make sure previous row and next row data is also fetched if its not fetched
+      if(scrollInBetween && this.scrollbarV && this.virtualization && this.externalPaging) {    
+        const upRow = this.rows[this.indexes.first - 1];
+        if(!upRow && direction === 'up')
```

Comment:
do not use if without brackets. run a ng lint --fix to align to eslint rules

Reply:
Fixed